templates/CRM/Form/validate.tpl

```diff
@@ -9,7 +9,7 @@
 *}
 {* Initialize jQuery validate on a form *}
 {* Extra params and functions may be added to the CRM.validate object before this template is loaded *}
-{if !$crm_form_validate_included and $smarty.get.snippet neq 'json' and $form and $form.formClass}
+{if empty($crm_form_validate_included) and !empty($smarty.get.snippet) and $smarty.get.snippet neq 'json' and !empty($form) and !empty($form.formClass)}
```

Comment:
@seamuslee001 I think this line isn't intended to change the behavior of the `if` statement, but it does - this prevents `crmValidate()` from being loaded on event registration pages:

```
and !empty($smarty.get.snippet)
```

This change causes the conditional to fail on event registration pages where previously it succeeded.  I haven't found a scenario where that directly affects core, but it breaks CiviHoneypot and I wouldn't be surprised if there were configurations that could break core as well.

Reply:
@MegaphoneJon you are right - this looks like an unintentional change